blocks/api/serializer.js

```diff
@@ -31,20 +32,21 @@ export function getBlockDefaultClassname( blockName ) {
  * Given a block type containg a save render implementation and attributes, returns the
  * enhanced element to be saved or string when raw HTML expected.
  *
- * @param {Object} blockType  Block type.
- * @param {Object} attributes Block attributes.
+ * @param {Object} blockType   Block type.
+ * @param {Object} attributes  Block attributes.
+ * @param {?Array} innerBlocks Nested blocks.
  *
- * @returns {Object|string} Save content.
+ * @returns {Object|string} Save element or raw HTML string.
  */
-export function getSaveElement( blockType, attributes ) {
+export function getSaveElement( blockType, attributes, innerBlocks = [] ) {
 	const { save } = blockType;
 
 	let saveElement;
 
 	if ( save.prototype instanceof Component ) {
```

Comment:
Unrelated to this PR: Is there any valid reason to still support "Component" in the save method? It creates the illusion that you can use lifecycle and state but you can't in reality.

Reply:
Would be good to follow up on this separately.

Reply:
One reason might be reusability of a same component between `edit` and `save`, though this sounds unlikely. I'd argue we shouldn't necessarily _prevent_ the use of a component, but similarly it would be nice if we could not target it explicitly; which is to say, the problem here isn't that we're checking `Component`, it's that below we call it as a function. In both the class and function component, I think we should be using `createElement`.

Reply:
Why should we treat it as a React Component when we don't support it being a React Component. It's just a function in all cases?